sa/model.go

```diff
@@ -383,6 +384,17 @@ type orderModel struct {
 	BeganProcessing   bool
 }
 
+type orderModelv2 struct {
+	ID                     int64
+	RegistrationID         int64
+	Expires                time.Time
+	Created                time.Time
+	Error                  []byte
+	CertificateSerial      string
+	BeganProcessing        bool
```

Comment:
I _think_ doing this would work:
```suggestion
	orderModelv1
```

Reply:
I would have to wrap those fields inside of `orderModelv1: orderModelv1{ID: blah}` and chose not to do that. Good suggestion though.